core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -80,6 +80,49 @@ var ExplorationPlayerPage = function() {
   var suggestionPopupLink =
     element(by.css('.protractor-test-exploration-suggestion-popup-link'));
 
+  var audioBar = element(
+    by.css('.protractor-test-audio-bar'));
+  var audioSelect = element(
+    by.css('.audio-language-select'));
+
+  this.clickAudioBar = function() {
+    waitFor.elementToBeClickable(audioBar,
+      'Audio bar taking too long to be clickable');
+    audioBar.click();
+  };
+
+  this.playAudio = function() {
+    var playBtn = element(
+      by.css('.fa-play-circle'));
```

Comment:
Could we use a protractor class for this button? Ditto for all the other buttons.

Reply:
done!